control/tests/freqresp_test.py

```diff
@@ -448,7 +448,7 @@ def test_dcgain_consistency():
     np.testing.assert_equal(
         sys_tf(0j, warn_infinite=False), complex(np.nan, np.nan))
     np.testing.assert_equal(
-        sys_tf.dcgain(warn_infinite=False), complex(np.nan, np.nan))
+        sys_tf.dcgain(warn_infinite=False), np.nan)
 
```

Comment:
Consider adding a test to show that dcgain() on a system with negative gain at zero frequency returns a negative number.

Reply:
Already exists:

https://github.com/python-control/python-control/blob/8c9e807ec64c62cde1dc6e174580099dfac8afd9/control/tests/freqresp_test.py#L505-L507